### ISSUE TITLE
The attention mask and the pad token id were not set

### ISSUE THREAD

**Issue by CanvaChen** [QwenLM/Qwen-7B#63]
user：Who are you?
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151643 for open-end generation.
A decoder-only architecture is being used, but right-padding was detected! For correct generation results, please set `padding_side='left'` when initializing the tokenizer.
assistant：A．

A<|im_end|>

**Comment by Modas-Li** [comment]
same issue
